SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLSubscribeVehicleDataSpec.m

```diff
@@ -16,7 +16,7 @@
 
 describe(@"Getter/Setter Tests", ^ {
     it(@"Should set and get correctly", ^ {
-        SDLSubscribeVehicleData* testRequest = [[SDLSubscribeVehicleData alloc] init];
+        SDLSubscribeVehicleData* testRequest = [SDLSubscribeVehicleData new];
```

Comment:
We do not use `new`; stick with `alloc init`

Reply:
OK, I will